crates/bevy_render/src/camera/camera_driver_node.rs

```diff
@@ -53,10 +54,14 @@ impl Node for CameraDriverNode {
             }
         }
 
+        // For the first several frames, no pipelines have finished compiling, and so no work will have been sent to the GPU,
+        // causing error messages on startup. Scheduling some dummy work bypasses this.
+        let first_several_frames = world.resource::<FrameCount>().0 <= 100;
```

Comment:
Can we have a FIXME here? I won't stand in the way because this is a big win but we should have an issue on file to fix this properly.

Reply:
```suggestion
        let any_creating = world
            .resource::<PipelineCache>()
            .pipelines()
            .any(|pipeline| matches!(pipeline.state, CachedPipelineState::Creating(_)));

[...]

                if camera_windows.contains(id) && !any_creating {
```
bit expensive maybe - could be calculated and stored in the pipeline cache when pipelines are queued or complete

Reply:
actually don't do my suggestion ... it would behave badly with `ClearColorConfig::None` cameras whenever a new pipeline gets queued...

i think we really just want to wait for the upscaling pipeline to compile, but that lives in `bevy_core_pipeline`... so would need some resource in bevy_render that core pipeline can modify to tell us what pipeline to wait for. then just don't run the camera graph until it's ready.

Reply:
new suggestion
```rust
    /// Block on compilation of a pipeline
    #[inline]
    pub fn block_on_render_pipeline(&mut self, id: CachedRenderPipelineId) {
        if self.pipelines.len() <= id.0 {
            self.process_queue();
        }
        let state = &mut self.pipelines[id.0].state;
        if let CachedPipelineState::Creating(task) = state {
            *state = match bevy_tasks::block_on(task) {
                Ok(p) => CachedPipelineState::Ok(p),
                Err(e) => CachedPipelineState::Err(e),
            };
        }    
    }
```
and block on the upscaling pipeline